external/corefx-bugfix/src/Common/src/System/Collections/Generic/LargeArrayBuilder.cs

```diff
@@ -234,7 +234,7 @@ public CopyPosition CopyTo(CopyPosition position, T[] array, int arrayIndex, int
             T[] buffer = GetBuffer(row);
             int copied =
 #if __MonoCS__
-            CopyToCore(buffer, column, array, arrayIndex, count);
+            CopyToCore(buffer, column, array, ref arrayIndex, ref count);
```

Comment:
This define is just for Mono compiler. Do we only need this when building with mcs?